### ISSUE TITLE
A simple Question about SMD Dataset

### ISSUE THREAD

**Issue by Garson-hu** [smallcowbaby/OmniAnomaly#42]
Thanks for you disclose SMD dataset.
I want know why former half part of dataset for training don’t have labels? Does this mean that all data in train dataset is normal？

**Comment by GPla** [comment]
Yes, the training data is expected to be normal behavior only. However, there can still be a few outliers.

**Comment by SnowZhao-wazi** [comment]
Can we have the label of training set also?
